pkg/edgecache/cachevolume/pvc_annotator.go

```diff
@@ -41,7 +42,8 @@ const (
 )
 
 var (
-	validStorageAuthentications = []string{"WorkloadIdentity", "AccountKey"}
+	validStorageAuthentications      = []string{"WorkloadIdentity", "AccountKey"}
+	ErrVolumeAlreadyBeingProvisioned = errors.New("pv is already being provisioned")
```

Comment:
super minor, but is this const() rather than var?

Reply:
i thought it wasn't possible, but ill double check

Reply:
yeee go mad
`(value of type error) is not constant`

Reply:
cool, thanks for checking! resolved.